### PR TITLE
fix Top Collection buttons

### LINKED ISSUE #10267: Top Collection buttons got reduced
[Gravação de tela de 09-05-2024 12:07:42.webm](https://github.com/kodadot/nft-gallery/assets/36627808/080aaf60-ba4f-4beb-b915-eb67e3771ecf)

### PR DESCRIPTION
**Pull request by FG-HJ**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

(please remove design and QA checks below if not needed)

## Needs Design check

- @exezbcz please review

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #10267 
- [ ] Requires deployment <worker/stick/speck>

#### Before submitting pull request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ahp/collection> 
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [ ] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=<My_Polkadot_Address_check_https://github.com/kodadot/nft-gallery/blob/main/REWARDS.md#creating-your-dot-address>)

#### Community participation

- [ ] [Are you at Koda Ecosystem Telegram?](https://t.me/koda_eco)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by prury** [comment]
i think changing the neobutton css directly affected too many items across app, can you come up with a different solution?

**Review by hassnian — CHANGES_REQUESTED** [review]
this change affects the entire app

as said here  https://github.com/kodadot/nft-gallery/issues/10267#issuecomment-2105606883

you need to override NeoButton's styles where that component is used, in this case TopCollections.vue

**Comment by hassnian** [comment]
@FG-HJ you requested a new review but I don't see any new changes , forgot to push ?

**Review by hassnian — CHANGES_REQUESTED** [review]
`NeoButton.scss` shouldn't be changed at all 

just override the min-w  of`NeoButton.scss` inside `TopCollections.vue`

File: components/landing/topCollections/TopCollections.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
         <NeoButton
           v-for="{ value, label } in timeRanges"
           :key="value"
-          class="control flex-1 p-0 min-w-[70px] max-md:w-full md:min-w-[98px] px-4 mobile-padding"
+          class="!min-w-[70px] md:!min-w-[110px] control flex-1 p-0 max-md:w-full px-4 mobile-padding"
```
Comment (hassnian) [inline components/landing/topCollections/TopCollections.vue:13]:
```suggestion
          class="control flex-1 p-0 !min-w-[70px] max-md:w-full md:!min-w-[98px] px-4 mobile-padding"
```

why did you change `md:!min-w-[110px]` width amount ?

**Comment by prury** [comment]
@FG-HJ there should not be changes in the pnpm-lock.yaml file in your PR

**Pull request closed without merging**